apps/backoffice/mocks/data/backend-data.ts

```diff
@@ -303,12 +308,20 @@ export const getMockServiceHistory = (
             : new Date(a.last_update).getTime() -
               new Date(b.last_update).getTime()
           : new Date(b.last_update).getTime() -
-            new Date(a.last_update).getTime()
+            new Date(a.last_update).getTime(),
       ),
-    continuationToken: faker.helpers.arrayElement([
-      faker.string.alphanumeric({ length: { min: 5, max: 10 } }),
-      undefined
-    ])
+  };
+};
+
+export const getMockInstitutionGroups = (institutionId?: string) =>
+  Array.from({ length: 3 }, () => ({
+    id: faker.string.uuid(),
+    name: faker.lorem.word(),
+  }));
+
+export const getUpsertManageGroupSubscriptionResponse = (groupId?: object) => {
+  return {
+    groupId,
   };
```

Comment:
**chore**: incorrect result type.

[200 response content](https://github.com/pagopa/io-services-cms/blob/master/apps/backoffice/openapi.yaml#L1014) of `upsertManageGroupSubscription` API Operation is a [Subscription schema definition](https://github.com/pagopa/io-services-cms/blob/master/apps/backoffice/openapi.yaml#L1585), which has two required properties: id and name.